weaviate_cli/managers/data_manager.py

```diff
@@ -72,6 +72,13 @@ def __import_json(
                         )
                     return -1
 
+                if self.client.batch.failed_objects:
```

Comment:
But this is only required if the batch would be of type client, but in this case it's a collection batch (one example would be, adding objects to a collection which doesn't exist and let auto-schema handle the schema, as there is not a collection created you need to use the self.client.batch)

Reply:
Ok! Understood. Removed. Thanks for the review :)